Week1/Swarnima_Shishodia/Que12.cpp

```diff
@@ -0,0 +1,61 @@
+#include <bits/stdc++.h>
+using namespace std;
+void simpleSieve(int limit, vector<int> &prime)
+{
+    bool mark[limit+1];
+    memset(mark, true, sizeof(mark));
+    for (int p=2; p*p<limit; p++)
+    {
+        if (mark[p] == true)
+        {
+            for (int i=p*2; i<limit; i+=p)
+                mark[i] = false;
+        }
+    }
+    for (int p=2; p<limit; p++)
+    {
+        if (mark[p] == true)
+        {
+            prime.push_back(p);
+            cout << p << " ";
+        }
+    }
+}
+
+void segmentedSieve(int n)
```

Comment:
Add comment what this function is doing and a small approach that you are trying to implement